pyprof/nvtx/nvmarker.py

```diff
@@ -41,6 +41,10 @@
 import math
 import json
 
+# Flag to indicate if traceMarker should collect a func_stack or not
+#
+enable_func_stack = False
```

Comment:
Can we not use globals.

Reply:
We should use a config / flags object (right now with just one member) e.g. `config.enable_func_stack` and use that instead.

Reply:
Sure. Is there something built in to python that does this or do I create a new class?

Reply:
I don't know (may be). For now we can just create a new class `Config` to hold all the flags.

Reply:
Should that be in a new file? Do I have to pass it around into all of the functions that need it or is the class globally accessible?

Reply:
We can create a new file for the config class. For now, I think its ok if the instance of the class is globally accessible. Ideally it should be a singleton class, so any function can call it and get/set the same object and we would not need a global variable.

Reply:
Thanks for the feedback. I pushed in some changes. I'm new to python so let me know if I didn't do some things correctly.